client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -92,6 +92,7 @@ type PlanFeatures2023GridProps = {
 	domainName: string;
 	placeholder?: string;
 	isLandingPage?: boolean;
+	intervalType: string;
```

Comment:
Since there will be fixed values for `intervalType`, can we tighten up this type to something like 
```
intervalType: 'monthly' | 'yearly';
```

The type could be moved to a common place so that it can be used across the codebase.

Reply:
I tried this but it would enforce a cascading change across multiple files. I will see if I can localize this change.